### PR TITLE
Handle changes to the `accept` attribute during the drag

### PR DESCRIPTION
**Pull request by Tom-Bonnike**
Open source in a nutshell, eh? 😄 #427 
I wasn’t sure how to deal with: `I'll deprecate classNames soon and this will cause troubles in the future. Could we check props on children?`.
I saw that it was done this way in another test so I just copied it.

### CODE REVIEW

**Comment by Tom-Bonnike** [comment]
Oh, didn’t see I needed to do the test change elsewhere. Sorry

File: src/index.spec.js [hunk 1]
```diff
@@ -332,44 +336,68 @@ describe('Dropzone', () => {
...
+          {({ isDragActive, isDragReject }) => {
+            if (isDragReject) {
+              return 'Rejected';
+            }
+            if (isDragActive) {
+              return 'Active';
+            }
+            return 'Empty';
+          }}
+        </Dropzone>
```
Comment (Tom-Bonnike) [inline src/index.spec.js]: Is this fine?

Comment (okonet) [inline src/index.spec.js]: I'm wondering if we could just render some simple component with those props and then in the test check if it received the proper props. I think this is what we should test not the classNames — they are just implementation detail.

Comment (okonet) [inline src/index.spec.js]: Somrthing like

```jsx
const MyComponent = () => null
```

so you can query it with enzyme.

Comment (Tom-Bonnike) [inline src/index.spec.js]:
```javascript
const DummyChildComponent = () => null;

const dropzone = mount(
  <Dropzone {...classConfig}>
    {props => <DummyChildComponent {...props} />}
  </Dropzone>
);
dropzone.simulate('dragEnter', { dataTransfer: { files } });
expect(dropzone.find(DummyChildComponent).props().isDragActive).toEqual(true);
expect(dropzone.find(DummyChildComponent).props().isDragReject).toEqual(false);
```
Like this?

Comment (okonet) [inline src/index.spec.js]: https://github.com/blainekasten/enzyme-matchers#tohaveproppropkeystring-propvalueany

File: testSetup.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 /* eslint prefer-template: 0 */
+import 'jest-enzyme';
```
Comment (Tom-Bonnike) [inline testSetup.js]:
```
Missing file extension for "jest-enzyme" (import/extensions)
import 'jest-enzyme';
```

? 👀

Comment (okonet) [inline testSetup.js]: That's weird. I'm checking this out.

Comment (okonet) [inline testSetup.js]: Try to get latest master or this commit: 4af134583be5eab3adb898f879879c25fa7ee517

**Comment by okonet** [comment]
Thanks again for your work on this! Appreciate that you jumped in to it.

**Pull request merged**